day-to-day.md

```diff
@@ -41,6 +41,14 @@ In order for us to fullfil our obligations to clients and to best work with your
 Try to have the day's hours in and rounded to the quarter hour by the end of the day. If we feel like we have a lot of billable work but aren't tracking a high number of billable hours, that
 will likely trigger a conversation around team productivity.
 
+### Time off requests
```

Comment:
This is a question that comes up in onboarding, and I realized we don't really have anything addressing it here.